src/Symfony/Component/Console/Tester/TesterTrait.php

```diff
@@ -29,45 +29,29 @@
     /**
      * Gets the display returned by the last execution of the command or application.
      *
-     * @param bool $normalize Whether to normalize end of lines to \n or not
-     *
      * @return string The display
      */
-    public function getDisplay($normalize = false)
+    public function getDisplay()
```

Comment:
this feature should be deprecated, not removed, as some code might still be using `PHP_EOL` when writing messages, and so this logic might still be needed for a full migration.